src/app/partitions/index.component.ts

```diff
@@ -182,6 +183,10 @@ app-table-actions-toolbar {
     TasksIndexService,
     FiltersService,
     PartitionsFiltersService,
+    {
+      provide: DATA_FILTERS_SERVICE,
+      useClass: PartitionsFiltersService
```

Comment:
since we already inject `PartitionsFiltersService`, we will prefer to use `useExisting`.

https://stackoverflow.com/questions/45152995/useclass-vs-useexisting
```suggestion
      useExisting: PartitionsFiltersService
```